merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/GetSimulationResultsAction.java

```diff
@@ -97,9 +100,13 @@ public Map<String, List<Violation>> getViolations(final PlanId planId, final Sim
       throw new RuntimeException("Assumption falsified -- mission model for existing plan does not exist");
     }
 
+    final var _results = this.simulationService.get(planId, revisionData, Function.identity());
```

Comment:
Now *this* is where I'd use `results$` 😉

Reply:
Change has been made!